src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+import duke.exception.DukeException;
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Encapsulates the information and operations of a task list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Deletes a task specified by task id from the task list.
+     * @param taskId The task id of the task which will be deleted.
+     * @return The task which is deleted.
+     * @throws DukeException Exception if there is error when deleting the task.
+     */
```

Comment:
Do note that there should be an empty line between the description and the parameters. I noticed the same issue in your other JavaDoc comments as well. Perhaps consider configuring your IDE to help with this styling?